umh-core/pkg/gatekeeper/certificatehandler/handler.go

```
@@ -0,0 +1,347 @@
+// SetSubHandler sets the subscriber handler for resolving who to fetch certs for.
+func (h *CertHandler) SetSubHandler(sh SubHandler) {
+	h.mu.Lock()
+	defer h.mu.Unlock()
+	h.subHandler = sh
+}
+
+// HasSubHandler returns true if the subscriber handler is available.
+func (h *CertHandler) HasSubHandler() bool {
+	h.mu.RLock()
+	defer h.mu.RUnlock()
+	return h.subHandler != nil
```

golangci-lint (2.12.2)

[error] return with no blank line before

(nlreturn)